### DISCUSSION TITLE
Route params return "()" parentheses after the param name

### DISCUSSION THREAD

**Discussion by lucassimines** [vuejs/router#1807] [Help and Questions] [answered]
I experienced a change of the `param` string when calling the `route.children`.

Using an `id` as a param, the path used to return only `:id`, and now it's returning `:id()`, is this some update or maybe an issue to be fixed?

![Screenshot 2023-04-19 at 13 38 46](https://user-images.githubusercontent.com/12851638/233143618-6bc1ffe9-9f4d-422f-a667-a1de50c9b15f.png)

**Comment by posva** [comment, marked as answer]
I think Nuxt does this by default to improve the support of mixed params routes: file `[a]_b` becomes `:a()_b` instead of `:a_b` which would have a param named `a_b`. This is a valid syntax for params.
